ludwig/features/feature_utils.py

```diff
@@ -100,7 +100,11 @@ def sanitize(name):
 
 
 def compute_feature_hash(feature: dict) -> str:
-    preproc_hash = hash_dict(feature.get(PREPROCESSING, {}))
+    preproc = feature.get(PREPROCESSING, {})
```

Comment:
Let's update the type hint and add a comment/docstring about when feature is a dict and when it's a schema object.

Reply:
Ah good eye, I remember noticing that during the call but pretty quickly forgot. Thanks for pointing that out. Will change now!